### PR TITLE
Support homebrew on linux with formula

### PR DESCRIPTION
**Pull request by rothgar**
Update adds support for `brew install` on Linux.

You may also want to look at using https://goreleaser.com/ to simplify some of the complexity in managing a custom script.

### CODE REVIEW

File: tasks [hunk 1]
```diff
@@ -124,24 +124,36 @@ const DojoVersion = \"${next_version}\"
         ;;
     homebrew_tap)
         if [ ! -f bin/dojo_darwin_amd64 ]; then echo "dojo_darwin_amd64 binary does not exist"; exit 1; fi
-        SHA256=$(sha256sum bin/dojo_darwin_amd64 | awk '{printf $1}')
+        if [ ! -f bin/dojo_darwin_amd64 ]; then echo "dojo_linux_amd64 binary does not exist"; exit 1; fi
```
Comment (tomzo) [inline tasks:127]: I think this should be `bin/dojo_linux_amd64`

**Review by tomzo — CHANGES_REQUESTED** [review]
Hi @rothgar 

Thanks a lot for the contribution.

I appreciate the suggestion - we'll look into goreleaser, it might make the whole ./tasks files shorter.

File: tasks (hunk 1 above)
Comment (rothgar) [reply tasks:127]: Good catch! I'll update it.

**Comment by rothgar** [comment]
Here's an example of goreleaser for a project I worked on in the past.

https://github.com/disneystreaming/ssm-helpers/blob/v1/.goreleaser.yml#L36

It'll automatically make Linux friendly formula and also verify x86 vs arm. Goreleaser also has some nice ways to cross compile binaries.

https://github.com/disneystreaming/homebrew-tap/blob/master/Formula/ssm-helpers.rb

**Pull request merged**